calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
```

Comment:
Can we define the calculateAverage function in a different way? Do we need to save the function as a const variable?

Reply:
This looks great and we could  also add a docstring to describe input, outpu of function. Good job!